presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -415,7 +311,7 @@ as a part of SQL query by appending name with the table.
        0     | s3://my-bucket/ctas_nation/data/9f889274-6f74-4d28-8164-275eef99f660.parquet | PARQUET     |           25 |               1648 | {1=52, 2=222, 3=105, 4=757} | {1=25, 2=25, 3=25, 4=25} | {1=0, 2=0, 3=0, 4=0} |  NULL            | {1=0, 2=ALGERIA, 3=0, 4= haggle. careful} | {1=24, 2=VIETNAM, 3=4, 4=y final packaget} | NULL         | NULL          | NULL
 
 Time Travel
-------------------------
+-----------
```

Comment:
Shall we add a section for SELECT support before Time travel? We can explain what type of Iceberg queries are supported by Presto there, e.g. read optimized, delta, real-time. We can also explain the support of COW and MOR tables there, or before talking about table properties

Reply:
@yingsu00 I have added the `SELECT` section in the `SQL Support` section. Since currently  Iceberg format version 1 & version 2 both read support is there but with limitations on version 2 tables where Delete files are present. I have mentioned that limitation as well. Please let me know if we should add any further details there.